spec/models/user_spec.rb

```diff
@@ -8,8 +8,14 @@
   it {should validate_length_of(:password).on(:create)}
   it {should validate_presence_of(:password).on(:create)}
   it {should have_secure_password}
-  it {should have_many(:reviews)}
+  it {should have_many(:reviews).order("created_at DESC")}
   it {should have_many(:queue_items).order("position")}
+  it {should have_many(:following_relationships)}
+  it {should have_many(:leading_relationships)}
+
+   user2 = Fabricate(:user, first_name: "Jane", last_name: "Doe")
+   user3 = Fabricate(:user)
+   friendship1 = Fabricate(:friendship, leader: user3, follower: user2)
```

Comment:
You are creating objects outside of `let`, `it` or `before` blocks. These will pertain in the database after you run RSpec. Use `let`.